typespec/Applications.Core/secretstores.tsp

```diff
@@ -47,7 +47,7 @@ model SecretStoreResource is TrackedResourceRequired<SecretStoreProperties, "sec
 
 @doc("The properties of SecretStore")
 model SecretStoreProperties {
-  ...ApplicationScopedResource;
+  ...GlobalScopedResource;
```

Comment:
Can you please add SecretStores_CreateOrUpdate_GlobalScope.json example like below ?

https://github.com/radius-project/radius/blob/main/typespec/Applications.Core/examples/2023-10-01-preview/SecretStores_CreateOrUpdate.json

Overall LGTM :)